ui/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "dependencies": {
     "@fortawesome/free-solid-svg-icons": "^5.15.2",
+    "autoprefixer": "^10.2.5",
```

Comment:
Any reason you added this ?

Reply:
Nope! It was unintended I have not used it. I will remove it asap!